library/Fission/Email/SendInBlue/Request/Types.hs

```diff
@@ -0,0 +1,23 @@
+module Fission.Email.SendInBlue.Request.Types (Request(..)) where
+
+import           Fission.Prelude
+
+import           Fission.Email.Recipient.Types
+import           Fission.Email.SendInBlue.TemplateOptions.Types
+import           Fission.Email.SendInBlue.TemplateId.Types
+
+import qualified Data.Vector as Vector
+
+
+data Request = Request
+  { templateId :: !TemplateId
+  , to         :: ![Recipient]
+  , params     :: !TemplateOptions
+  } 
+
+instance ToJSON Request where
+  toJSON Request { templateId = templateId', to = to', params = params' } =
```

Comment:
`toJSON Request { templateId, to, params } = ...`

(No need for the `=` and `’`)